src/components/MetaData.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+  export let data;
+</script>
+
+{#if data}
+  <div
+    class="inline-block border border-blue-500 rounded-full text-xs py-1 px-3 w-15 bg-blue-800 text-white">
+    Deprecated:
+    {data.deprecated}
```

Comment:
This content should be set by the parent component. See the svelte documentation if you're not sure how to do this.